server/src/main/java/org/apache/druid/client/cache/Cache.java

```diff
@@ -46,6 +48,9 @@
 
   void close(String namespace);
 
+  @Override
+  default void close() throws IOException {}
```

Comment:
Why a default def is needed here?

Reply:
Some cache implementations may not need to do some work when process exit. Adding a default def, so they can implement it when necessary, and also for code compatibility.

Reply:
I don't see Cache is a PublicApi. Also if some Cache doesn't need to add close(), I think it's better that it explicitly adds the method with an empty body.

Reply:
Hmm, ok I'll modify it.